docs/advanced-features/output-file-tracing.md

```diff
@@ -101,15 +101,8 @@ module.exports = {
       // if there is `process.cwd()` expression in your code, you can set this option to tell `turbotrace` the value of `process.cwd()` while tracing.
       // for example the require(process.cwd() + '/package.json') will be traced as require('/path/to/cwd/package.json')
       processCwd?: string
-      // control the maximum number of files that are passed to the `turbotrace`
-      // default is 128
-      maxFiles?: number
-      // control the maximum memory usage of the `turbotrace`, in `MB`
+      // control the maximum memory usage of the `turbotrace`, in `MB`, default is `6000`.
       memoryLimit?: number
-      // control if the webpack entries should be skipped when tracing
-      // tracing the entries is not necessary for applications that all assets could be traced in the dist files
-      // default is `false`
-      skipEntries?: boolean
```

Comment:
If these additional properties are passed in, will it error or silently ignore?

Reply:
ignore